Dockerfile

```diff
@@ -1,17 +1,40 @@
-FROM python:3.7
+FROM ubuntu:eoan
```

Comment:
Did you test running a Radiance command? I wonder if the Radiance build will work fine even if it has been built against a different version of ubuntu.

Reply:
`rtrace` worked fine for me. I don't foresee any issues between ubuntu versions. Had it been compiled for a completely different arch like windows or macos I would be more concerned.

In principle it should work and my small tests indicate it does. We can carry out more tests by running workflows to determine whether the image base is suitable.